examples/saas-dynamic-dashboards/frontend/app/Client/client.ts

```
@@ -0,0 +1,10 @@
+import axios, { AxiosRequestConfig } from "axios";
+
+export const client = async (config: AxiosRequestConfig) => {
+    try {
+        const res = await axios(config)
+        return res.data
+    } catch (error) {
+        throw error
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)